ios/Mattermost/UIImage+vImageScaling.h

```diff
@@ -0,0 +1,16 @@
+//
+//  UIImage+vImageScaling.h
+//  UIImage+vImageScaling
+//
+//  Created by Matt Donnelly on 03/07/2013.
+//  Copyright (c) 2013 Matt Donnelly. All rights reserved.
```

Comment:
Where does this come from? Do we need to add something to the license to this?

Reply:
credit is already granted in the header of the file right?